.github/workflows/reusable-build-and-push.yml

```diff
@@ -55,13 +60,20 @@ jobs:
           tags: ghcr.io/openclarity/kubeclarity:${{ inputs.image_tag }}
           file: Dockerfile.backend
           push: ${{ inputs.push }}
+          outputs: type=docker,dest=/tmp/kubeclarity.tar
```

Comment:
need to verify that `push: true` and `outputs` works well together since `push is a is a shorthand for --output=type=registry`

![image](https://user-images.githubusercontent.com/40820488/169785539-608b96ad-0a80-4f67-900a-81a3e6cd27a4.png)
from https://github.com/docker/build-push-action


from the code seems like it will fail
https://github.com/docker/buildx/blob/master/commands/build.go#L183

Maybe you can set an env var for output only if `upload` is true and `push` is false and use that env var here. Using and empty env var here might solve it.

example
![image](https://user-images.githubusercontent.com/40820488/169787076-98fb4af5-0909-4405-bbea-01bb26307855.png)